index.yaml

```diff
@@ -3,14 +3,14 @@ entries:
   admission-controller:
   - apiVersion: v2
     appVersion: 0.0.1
-    created: "2021-12-07T15:49:46.489022-06:00"
+    created: "2021-12-08T13:26:28.593417-06:00"
     dependencies:
     - condition: proxy-scanner.enabled
       name: proxy-scanner
       repository: https://lacework.github.io/helm-charts
       version: 0.2.2
     description: Lacework admission controller using Lacework proxy scanner
-    digest: 74ea4088ecb266b6a234ca108079e0f067541886823756893d3a02d68fe3423a
+    digest: d26d65d3650a8ff6b1b09859edbcbc9d11f7d4903a8e9b64c98b36ad2eea70d3
```

Comment:
I think we should bump the version on the charts too. I'm not sure how Helm handles charts that have the same version and conflicting digests.

Reply:
Is a new version needed? I've updated the package as well, so this digest should match that.

Reply:
I'm worried that Helm will reject the new package because the version is the same and the digest changed.

Reply:
Ah I see, I've bumped up the version in this case then.